src/pages/Login/Login.scss

```
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap');
+
+.loginWrap {
+  text-decoration: none;
+  color: inherit;
+  font-family: 'Nanum Gothic', sans-serif;
+  .loginHeader {
+    height: 55px;
+    .loginHeaderBox {
+      position: fixed;
+      width: 100%;
+      height: 55px;
+      border-bottom: 1px solid #dfdfdf;
+      z-index: 100;
+      border-bottom: 1px solid #f0f0f0;
+      box-shadow: 0 0 30px 0 rgb(255 255 255 / 90%);
+      .loginHeaderInner {
+        margin: 0 auto;
+        max-width: 768px;
+        position: relative;
+        h1.loginPageTitle {
+          line-height: 54px;
+          font-weight: 500;
+          font-size: 16px;
+          text-align: center;
+        }
+        button.loginClose {
```

className 부여하셨기 때문에 태그는 쓰지 않아도 됩니다!
```suggestion
        .loginPageTitle {
          line-height: 54px;
          font-weight: 500;
          font-size: 16px;
          text-align: center;
        }
        
        .loginClose {
```

---

리뷰 남기지 않은 다른 부분도 찾아서 수정해주세요!